raft/server.cc

```diff
@@ -501,6 +502,7 @@ future<> server_impl::add_entry(command command, wait_type type, seastar::abort_
         if (as && as->abort_requested()) {
             throw request_aborted();
         }
+        check_not_aborted();
```

Comment:
But we need it two places? I was thinking you will remove the one at the start of the function.

Reply:
I prefer thinking and not repeat checks after each line. If one check is enough it should not be repeated. Here the only difference I see is potentially different error returned it a message too large. This does not seams to be an issue. There is also consistency issue. Why do we check service abort at the entry but not request abort (`as->abort_requested()`). Should we also add it there?

Reply:
removed